app/mailers/views/rrd_completed_mailer.erb.html

```diff
@@ -8,12 +8,16 @@
     <p>
       A single-issue <%= @disability_struct[:label] %> (<%= @disability_struct[:code] %>) claim for increase was submitted on va.gov.<br/>
       <% case @rrd_status
-         when :processed %>
+         when 'processed' %>
         A health summary PDF was generated and added to the claim's documentation.
-      <% when :pending_ep %>
+      <% when 'pending_ep' %>
         There was already a pending EP 020 for the veteran associated with this claim.
-      <% when :insufficient_data %>
+      <% when 'insufficient_data' %>
         There was not sufficient data to generate a health summary PDF associated with this claim. 
+      <% when 'error' %>
+        There was an error with this claim: <%= @error_message %>
+      <% else %>
+        Metadata: <%= @submission.form.dig('rrd_metadata') %>
```

Comment:
We'll have to modify this if we ever put PII in `rrd_metadata`, which I don't foresee having to do.